src/Microsoft.AspNetCore.Sockets.Client.Http/LongPollingTransport.cs

```diff
@@ -23,7 +23,7 @@ public class LongPollingTransport : ITransport
         private Task _poller;
         private string _connectionId;
 
-        private readonly CancellationTokenSource _transportCts = new CancellationTokenSource();
+        private CancellationTokenSource _transportCts = new CancellationTokenSource();
```

Comment:
remove this completely - it is not being used.

Reply:
We actually don't want to be able to Cancel StartAsync in LongPolling though, right? The issue you described with outstanding polls

Reply:
We'd still need the _transportCts for the poll and send  but I think we just shouldn't create the linked cts with the Token passed into StartAsync?

Reply:
Yes. Longpolling would ignore the `CancellationToken` passed in `StartAsync`.